### DISCUSSION TITLE
Bug with disappearing fontawesome text in short text table column

### DISCUSSION THREAD

**Discussion by davhad** [AntlerVC/firetable#1310] [Support / Q&A] [answered]
A table has a short text column being used to store a fontawesome icon relevant to the record in question. When the icon character is added to the table either through the table or through the inspector view  and after waiting for the record to save (or by clicking anywhere) it shows as pasted and the new property is shown in the firebase record as seen through the console. 

When the rowy table is refreshed the icon property goes away and it also does the same from the firestore record.

Example icon:  
More info: https://fontawesome.com/icons/angel?f=classic&s=light

**Comment by BeeBombshell** [comment]
Hey @davhad, could you share a screen recording via https://www.jam.dev so that we can take a look? :)

**Reply by davhad** [reply, marked as answer]
This issue has stopped occurring for me right now so I wasn't able to record it anymore, however it's one of many such Rowy UI inconsistencies that I encounter habitually. I've opened two other discussions (both with dev.jam videos) one about not being able to create a new record using the main add row button. Have a feeling all of these things are related.

**Reply by BeeBombshell** [reply]
Thanks for the feedback, @davhad. I'll take a look.

**Comment by davhad** [comment]
I could after you confirm with me that you tried to recreate it? Just copy
the glyph in the link I shared with you into a text field and it should
happen to you too. Was a plain vanilla table.
